README.md

```diff
@@ -1,14 +1,25 @@
 # react-native-pusher-push-notifications
+
 Manage pusher interest subscriptions from within React Native JS
 
-IMPORTANT!!! This module is intended to complement the default [Pusher setup](https://pusher.com/docs/push_notifications).  This module simply allows that implementation to be accessed directly from React Native JS.
+**IMPORTANT!!!**
+
+This module is intended to complement the default [Pusher Carthage setup](https://docs.pusher.com/beams/ios/sdk-integration#install-from-carthage). This module simply allows that implementation to be accessed directly from React Native JS.
```

Comment:
I would rephrase this like:

This module is intended to complement the setup with [Carthage](https://pusher.com/docs/push_notifications).